src/currency.cpp

```diff
@@ -65,6 +65,7 @@ static const CurrencySpec origin_currency_specs[CURRENCY_END] = {
 	{   40, "", CF_NOEURO, "NTD" NBSP,     "",               0, STR_GAME_OPTIONS_CURRENCY_NTD    }, ///< new taiwan dollar
 	{    8, "", CF_NOEURO, "\xC2\xA5",     "",               0, STR_GAME_OPTIONS_CURRENCY_CNY    }, ///< chinese renminbi
 	{   10, "", CF_NOEURO, "HKD" NBSP,     "",               0, STR_GAME_OPTIONS_CURRENCY_HKD    }, ///< hong kong dollar
+	{   90, "", CF_NOEURO, "INR" NBSP,     "",               0, STR_GAME_OPTIONS_CURRENCY_INR    }, ///< Indian Rupee
```

Comment:
Would it make sense to use the ₹ symbol? I guess it would be anachronistic for earlier years, and might cause font issues...

(genuine question, I know nothing about how this is normally written in India)

Reply:
The ₹ is \u20B9 and the prefix part of the CurrencySpec is a char[]. So I don't think we can add the proper symbol. INR is fine for me personally.

Reply:
The "char[]" equivalent (or, UTF8) would be "\xE2\x82\xB9"

Reply:
> The "char[]" equivalent (or, UTF8) would be "\xE2\x82\xB9"

Can you tell me how this works or any link to it?

Reply:
OpenTTD uses the [UTF-8 encoding](https://en.wikipedia.org/wiki/UTF-8#Description), so non-ASCII characters can be represented using multiple bytes.

You can find the encoding for ₹ in [this table](https://www.utf8-chartable.de/unicode-utf8-table.pl?start=8352).